src/RL/representation/default/defaultstaterepresentation.jl

```diff
@@ -113,7 +113,7 @@ function update_representation!(sr::DefaultStateRepresentation, model::CPModel,
     update_features!(sr, model)
     ncon = sr.cplayergraph.numberOfConstraints
     nvar = sr.cplayergraph.numberOfVariables
-    sr.possibleValuesIdx = map(v -> indexFromCpVertex(sr.cplayergraph, ValueVertex(v)), collect(x.domain)) .- ncon .- nvar
+    sr.possibleValuesIdx = map(v -> indexFromCpVertex(sr.cplayergraph, ValueVertex(v)), collect(x.domain))
```

Comment:
I had to remove my change because it wasn't working anymore with `variableoutputcpnn` (index issue raised at [line 32](https://github.com/corail-research/SeaPearl.jl/blob/heterogeneous/src/RL/nn_structures/variableoutputcpnn.jl)).

I modified the function `from_order_to_id` consequently.

Reply:
Ok, no problem if it is adapted in `from_order_to_id`